OpenLIFUData/OpenLIFUData.py

```diff
@@ -258,6 +265,95 @@ def customexec_(self):
 
         return (returncode, volume_filepath,volume_name, volume_id)
 
+class AddNewPhotocollectionDialog(qt.QDialog):
+    """ Add new photocollection dialog """
+
+    MINIMUM_NUMBER_OF_PHOTOS_FOR_PHOTOSCAN=20
+
+    def __init__(self, parent="mainWindow"):
+        super().__init__(slicer.util.mainWindow() if parent == "mainWindow" else parent)
+        self.setWindowTitle("Add New Photocollection")
+        self.setWindowModality(qt.Qt.WindowModal)
+
+        self.reference_number = "".join(random.choices(string.digits, k=8))
+        self.pulled_files = []
+
+        temp_dir = tempfile.gettempdir()
+        self.temp_photocollection_path = os.path.join(temp_dir, self.reference_number)
+        os.makedirs(self.temp_photocollection_path, exist_ok=True)
+
+        self.setup()
+
+    def setup(self):
+
+        self.setMinimumWidth(400)
+        self.setContentsMargins(15, 15, 15, 15)
+
+        vBoxLayout = qt.QVBoxLayout()
+        vBoxLayout.setSpacing(10)
+        self.setLayout(vBoxLayout)
+
+        self.directionLabel1 = qt.QLabel(
+            f"Please create a 3D Open Water photocollection with the following reference number: {self.reference_number}. "
+            "Click \"OK\" when the Android device has finished and is plugged into the computer."
+        )
+        self.directionLabel1.setWordWrap(True)
+        vBoxLayout.addWidget(self.directionLabel1)
+
+        vBoxLayout.addStretch(1)
+
+        self.buttonBox = qt.QDialogButtonBox()
+        self.buttonBox.setStandardButtons(
+            qt.QDialogButtonBox.Ok | qt.QDialogButtonBox.Cancel
+        )
+        vBoxLayout.addWidget(self.buttonBox)
+
+        self.buttonBox.rejected.connect(self.reject)
+        self.buttonBox.accepted.connect(self.validateInputs)
+
+    def validateInputs(self):
+        """
+        We need to make sure that the android file system has the files
+        associated with the reference id in the right location.
+        """
+        # The path /sdcard/DCIM/Camera/ is the standard internal storage path
+        # from the Android device’s perspective when accessed via adb, not the
+        # computer’s mounted file system like with Android File Transfer.
+        android_dir = "/sdcard/DCIM/Camera"
+        result = subprocess.run(
+            ["adb", "shell", "ls", f"{android_dir}/{self.reference_number}_*.jpeg"],
+            capture_output=True, text=True
+        )
```

Comment:
Don't put a `/` in a string like that to construct a path, instead let pathlab construct the path so that it is os-independent

```suggestion
        android_dir = Path("/sdcard/DCIM/Camera")
        result = subprocess.run(
            ["adb", "shell", "ls", android_dir/f"{self.reference_number}_*.jpeg"],
            capture_output=True, text=True
        )
```